src/main/docs/guide/spring/springWithOpenApiView.adoc

```diff
@@ -0,0 +1,26 @@
+To use micronaut openapi views (Swagger UI, OpenAPi Explorer, Redoc, RapiDoc) you need to add static resources to Spring configuration like this:
+
+.WebConfig.java
+[source,java]
```

Comment:
can you use `snippet::` to include a real tested piece of code here?

Reply:
fixed